lib/std/testing.zig

```diff
@@ -642,6 +642,9 @@ pub fn tmpDir(opts: std.fs.Dir.OpenOptions) TmpDir {
     };
 }
 
+/// This functions is to be used only in tests. When the two strings are not
+/// equal, prints diagnostics to stderr to show exactly they are not equal,
+/// then returns a test expected equal error.
```

Comment:
its better to use name as it appears in code so its easier to parse visually "`error.TestExpectedEqual`" instead of just "test expected equal error".
```suggestion
/// then returns a `error.TestExpectedEqual`.
```
Also ticks ` allow IDEAs to understand what this is a symbol and its not just text.